mappings/net/minecraft/client/render/entity/model/ChestBoatEntityModel.mapping

```diff
@@ -2,3 +2,4 @@ CLASS net/minecraft/class_7752 net/minecraft/client/render/entity/model/ChestBoa
 	FIELD field_40467 CHEST_BOTTOM Ljava/lang/String;
 	FIELD field_40468 CHEST_LID Ljava/lang/String;
 	FIELD field_40469 CHEST_LOCK Ljava/lang/String;
+	METHOD method_45708 getChestBoatTexturedModelData ()Lnet/minecraft/class_5607;
```

Comment:
this is static, right? so it can be just `getTexturedModelData` to match other class' methods

Reply:
```suggestion
	METHOD method_45708 getTexturedModelData ()Lnet/minecraft/class_5607;
```